questionnaire/questionnaireUISchema.js

```diff
@@ -1673,6 +1673,9 @@ module.exports = {
             properties: {
                 'q-rep-has-reference-number': {
                     options: {
+                        macroOptions: {
+                            classes: 'govuk-radios'
+                        },
```

Comment:
Is this question changing schema? or was there a UI bug?

Reply:
UI bug